geoips/compare_outputs.py

```diff
@@ -741,7 +761,7 @@ def compare_outputs(compare_path, output_products, test_product_func=None):
 
     if len(goodcomps) > 0:
         fname_goodcptest = join(diffdir, "cptest_GOODCOMPARE.txt")
-        print("source {0}".format(fname_goodcptest))
+        LOG.info("source {0}".format(fname_goodcptest))
```

Comment:
DO NOT include any goodcompare stuff in print or LOG.interactive.  I don't care if they pass.. Just if they fail.